CHANGELOG.md

```diff
@@ -22,7 +22,7 @@ IMPROVEMENTS:
 
 BUG FIXES:
 * resource/vcd_vapp, resource/vcd_vapp_vm add vApp status handling when environment is very fast [GH-68]
-* resource/vcd_vapp_vm add additional validation to check if vApp template is OK [GH-157] 
+* resource/vcd_vapp_vm add additional validation to check if vApp template is OK [GH-151]
```

Comment:
Oops! 151 is a PR in go-vcloud-director project, so the above turned out linking to a wrong *issue* in terraform-provider-vcd

Reply:
Done. force pushed without the changes at all.